index.html

```diff
@@ -0,0 +1,109 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main id="main">
+          <h1 id="title">Dr Norman</h1>
+          <div id="img-div">
```

Comment:
Split your code into different section. You can not have only "main tag" in your html code. You split into sections like header, main, and footer. Also make use of other sematics tags like section, etc to make your code cleaner and easier to read.